src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/Level.java

```diff
@@ -1,13 +1,14 @@
 package com.minecolonies.coremod.entity.ai.citizen.miner;
 
 import com.minecolonies.coremod.colony.buildings.BuildingMiner;
+import com.minecolonies.coremod.util.Vec2i;
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.nbt.NBTTagList;
+import net.minecraft.util.math.MathHelper;
 import net.minecraftforge.common.util.Constants;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.jetbrains.annotations.Nullable'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)